pkg/archive/archive.go

```diff
@@ -530,7 +530,21 @@ func createTarFile(path, extractDir string, hdr *tar.Header, reader io.Reader, L
 		}
 
 	case tar.TypeLink:
-		targetPath := filepath.Join(extractDir, hdr.Linkname)
+		//ln a b in container
+		//docker cp test:/tmp tmp_
+		//hdr.Linkname = "tmp/a"
+		//hdr.Name = "tmp_/b"
+		//--> lpath = tmp_/a -->targetPath =extractDir + lpath --> ln tmp_/a tmp_/b
+		lpath := hdr.Linkname
+		flinkName := strings.Split(hdr.Linkname, "/")
```

Comment:
Should this always use `/`, or `os.PathSeparator`? I see other parts of the code use, e.g. `filepath.Join()`, which is platform dependent